web/gds-user-ui/src/components/Section/SearchDirectory.tsx

```diff
@@ -83,15 +98,32 @@ const SearchDirectory: React.FC<TSearchDirectory> = ({
               <form onSubmit={(e) => handleSubmit(e, search)}>
                 <FormControl color={'gray.500'}>
                   <HStack>
-                    <Input
-                      size="md"
-                      width={'100%'}
-                      type="search"
-                      isRequired
-                      placeholder={t`Common name or VASP ID`}
-                      name="search"
-                      onChange={(event: any) => setSearch(event.currentTarget.value)}
-                    />
+                    <AutoComplete
+                      rollNavigation
+                      // no found message
+                      noOptionsMessage="No VASP found">
+                      <AutoCompleteInput
+                        variant="filled"
+                        placeholder="Common name or VASP ID"
+                        autoFocus
+                      />
+                      <AutoCompleteList>
+                        {Object.keys(options)?.map((oid: any) => (
```

Comment:
For my understanding, does `oid` = `options ID`?

Reply:
> For my understanding, does `oid` = `options ID`?

Yes the oid is the key of option object.